collections/_posts/2023-09-15-SAM_for_Medical.md

```diff
@@ -0,0 +1,147 @@
+<p style="text-align: center;font-style:italic">Figure 6. Overview of SAMUS.</p>
+
+# Results
+
+As three of these methods were released in very a short space of time, they did not include cross-method comparison. Only SAMUS provide an evaluation of the four models on the same task.
+
+<div style="text-align:center">
+<img src="/collections/images/sam_for_med/medsam_results.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 7. Results of MedSAM on various tasks.</p>
+
+<div style="text-align:center">
+<img src="/collections/images/sam_for_med/samed_results.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 8. Results of SAMed on the Synapse multi-organ CT segmentation.</p>
+
+<div style="text-align:center">
+<img src="/collections/images/sam_for_med/msa_results.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 9. Results of MSA on multi-organ segmentation (AMOS dataset).</p>
+
+<div style="text-align:center">
+<img src="/collections/images/sam_for_med/samus_results.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 10. a) Results of SAMUS, comparison with state-of-the-art methods. b) Compirason of SAMUS with other SAM-adapted models.</p>
+
+&nbsp;
+
+Finetuning SAM on medical imaging dataset is essential to deal with the domain shift with natural images. However, more advanced stratagies, such as using low-rank adapter, seem to work better than finetuning all the parameters of the model. Adapted SAM reaches performances that are in-par or superior fully state-of-the-art supervised models on various tasks.
+
+Latest results shown in SAMUS tend to show that the adaptation strategy could be coupled with external feature extractor may also be useful to increase even more the performances of such  models.
+
+Other figures and additional results in the articles include ablation studies, results on more datasets as well as quantitative and qualitive results for multiple-point input prompts and/or box prompts.
+
+# Conclusion
+
+Several recent works have leveraged SAM architecture and training and tried to modify it in order to improve its capacities on medical imaging tasks. They mainly rely on Low-rank adaptation modules introduced inside the transformer's architecture of SAM, but new ideas, such as using a CNN as local features extractor, may leads to even an greater increase in performances in the future.
+
+# References
+[^1]: A. Kirillov, E. Mintun, N. Ravi,H. Mao, C. Rolland,L. Gustafson, T. Xiao, S. Whitehead, A. C. Berg,  W.-Y. Lo, P. Dollar, R. Girshick, *Segment Anything*, ICCV 2023
+[^2]: M. A. Mazurowski, H. Dong, H. Gu, J. Yang, N. Konz, Y. Zhang, *Segment Anything Model for Medical Image Analysis: an Experimental Study*, April 2023
+[^3]: E. J. Hu, Y. Shen, P. Wallis, Z. Allen-Zhu, Y. Li, S. Wang, L. Wang, W. Chen, *LoRA: Low-Rank Adaptation of Large Language Models*, ICLR 2022
```

Comment:
Same suggestion about adding hyperlinks directly to the papers (e.g. arXiv versions).

Reply:
done